doc/040_backup.rst

```diff
@@ -213,13 +213,26 @@ On most Unixy shells, you can either quote or use backslashes. For example:
 
 By specifying the option ``--one-file-system`` you can instruct restic
 to only backup files from the file systems the initially specified files
-or directories reside on. For example, calling restic like this won't
-backup ``/sys`` or ``/dev`` on a Linux system:
+or directories reside on. So it will prevent restic from crossing filesystem 
+boundaries when performing a backup. For example, if you backup ``/`` with this 
+flag but you have external media mounted under ``/media/usb`` then restic will 
+not back up ``/media/usb`` at all because it *is* a different filesystem.
+Also calling restic on ``/`` would not descend into any virtual
+filesystems so it won't backup ``/sys`` or ``/proc`` on a Linux system:
 
 .. code-block:: console
 
     $ restic -r /srv/restic-repo backup --one-file-system /
 
+Notice that this does not prevent you from specifying multiple filesystems on the command line, e.g:
+
+
+.. code-block:: console
+
+    $ restic -r /srv/restic-repo backup --one-file-system / /media/usb 
+
+will back up both ``/`` and ``/media/usb`` filesystems, but would not descend into virtual filesystems like ``/sys`` and ``/proc``.
```

Comment:
"will back up both" -> "will back up both the"

"but would not descend into virtual filesystems like" -> "but will not include other filesystems like"